temporalcli/commandsgen/commands.yml

```diff
@@ -3814,6 +3865,10 @@ commands:
           --workflow-id YourWorkflowId \
           --update-id YourUpdateId
       ```
+    options: 
+      - name: identity
```

Comment:
Is this not already available via the new available-to-all-clients `identity` arg?

Reply:
Fixed this everywhere